optuna/multi_objective/samplers/_nsga2.py

```diff
@@ -94,7 +94,7 @@ def __init__(
 
     def reseed_rng(self) -> None:
         self._random_sampler.reseed_rng()
-        self._rng = np.random.RandomState()
+        self._rng.seed()
```

Comment:
This is not strong opinion, but `optuna/multi_objective` module is deprecated, so we don't have to fix this modules.  (In mu understanding, if a feature is deprecated, it is under the maintenance mode, so we don't have to fix it unless it has a critical bug.)

Reply:
Thank you for your comment. I think you're right. Should I revert the change?

Reply:
> I think you're right. Should I revert the change?

Yes! It would be nice to revert them!

> By the way, should we replace...

Nice catch! Yes, I think it should be replaced, but don't have to use `multivariate=True`. (TPESampler automatically use multi-objective TPE if the number of objectives >= 2.)

And, perhaps a test could be added to `tests/samplers_tests/tpe_tests/test_multi_objective_sampler.py` to check that MOTPESampler is deprecated. It can be a followup.

Reply:
# Revert the deprecated samplers under `optuna/multi_objective/samplers/`

I realised that these samplers somehow inherit `optuna/samplers`, so we need to fix the tests because the current tests for these deprecated samplers failed by introducing the change to `optuna/samplers` by this PR.

# Replace `MOTPESampler ` with `TPESampler `

Good to know that. Thank you for your comment.